ev-dev-tools/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-ev_create_pip_install_targets(
+ev_pip_install_local(
```

Comment:
This is not really related to the commit title, therefore I would appreciate, if it would be mentioned at least in the commit message, why this has been done

Reply:
I'll add this in a different PR since it's not really related to these changes

Reply:
done in the PR instead: https://github.com/EVerest/everest-utils/pull/157